### PR TITLE
Add default labels to proto rules

### PR DESCRIPTION
**Pull request by fs75**

### CODE REVIEW

File: src/parse/rules/proto_rules.py [hunk 1]
```diff
@@ -20,6 +20,10 @@
 }
 
 
+DEFAULT_PROTO_LABELS = ['protobuf']
```
Comment (peterebden) [inline src/parse/rules/proto_rules.py]: Can you make these variables private please?

Comment (peterebden) [inline src/parse/rules/proto_rules.py]: We already have this, but it's called "proto" - try e.g. `plz query alltargets --include proto --hidden`. Adding grpc separately is probably useful though.

Comment (fs75) [inline src/parse/rules/proto_rules.py]: ok, removed protobuf
didn't know about proto

**Comment by peterebden** [comment]
Looks good, thanks Fabian!

**Pull request merged**